public/components/FeatureAnywhereContextMenu/AnywhereParentFlyout/AnywhereParentFlyout.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import './styles.scss';
```

Comment:
This can be removed, look at this comment: https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/482/files#r1195437480